pb_plugins/dcsdkgen/type_info.py

```diff
@@ -2,6 +2,24 @@
 from os import environ
 
 
+class TypeInfoFactory:
+
+    def __init__(self):
+        self._conversion_dict = self.__load_conversions_dict()
+
+    def create(self, field):
+        return TypeInfo(field, self._conversion_dict)
+
+    def __load_conversions_dict(self):
```

Comment:
`__` is usually an python internal function name, I'd suggest to go with just one `_`!

Reply:
`_` is private, and `__` internal then?

Reply:
Exactly! There is no such thing as private in python but it is the common way to do it over here :)